src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -560,9 +639,13 @@ private static Collection<ScriptBindingProvider> LoadBindingProviders(ScriptHost
                 typeof(NotificationHubScriptBindingProvider),
                 typeof(SendGridScriptBindingProvider),
                 typeof(TwilioScriptBindingProvider),
-                typeof(BotFrameworkScriptBindingProvider)
-            };
+                typeof(BotFrameworkScriptBindingProvider),
```

Comment:
I don't think so - we only know the model you are proposing actually works for bindings like all the existing ones if we actually try to replat them. If you have confidence in the new model, then it should be very easy to simply remove all these types. If it is not easy, the new model isn't working yet, and won't work for extensions people want to write.